guides/source/asset_pipeline.md

```diff
@@ -681,7 +681,11 @@ Capistrano (v2.15.1 and above) includes a recipe to handle this in deployment.
 Add the following line to `Capfile`:
 
 ```ruby
+# Capistrano version 2
 load 'deploy/assets'
+
+# Capistrano version 3
+require "capistrano/rails/assets"
```

Comment:
would this blow up for Cap v2?

Reply:
Sorry I cannot try it now. I'm glad if anyone try it.

Reply:
It does. Since we're mentioning a minimum version of `2.x` here I think it makes sense to have these two lines. 👍